README.md

```diff
@@ -31,6 +31,20 @@ It contains the following public packages:
 This repository also contains `gotpm`, a command line tool for using the TPM.
 Run `gotpm --help` and `gotpm <command> --help` for more documentation.
 
+### Use prebuilt `gotpm` binary
+
+You can download the binary from a [release](https://github.com/google/go-tpm-tools/releases) directly.
+
+```bash
+curl -L https://github.com/google/go-tpm-tools/releases/download/v0.4.3/go-tpm-tools_Linux_x86_64.tar.gz -o go-tpm-tools.tar.gz
```

Comment:
Use placeholder in the url ```https://github.com/google/go-tpm-tools/releases/download/[VERSION]/go-tpm-tools_[arch].tar.gz
`

Reply:
Perhaps just point to https://github.com/google/go-tpm-tools/releases ?